### ISSUE TITLE
Border left not display when using stroke and strokeWidth on SlicedChart push to Funnel Series

### ISSUE THREAD

**Issue by PongVL** [amcharts/amcharts4#3424]
**Bug description**
When I using the stroke and strokeWitdth properties of slicedChart to get the border lines on each sliced chart bar, I only see the top, bottom and right border line display, still missing the left border line.

My codepen javascripts: [https://codepen.io/pongvl/pen/MWJJRjE](https://codepen.io/pongvl/pen/MWJJRjE)

**Environment (if applicable)**
I'm using amchart4 core get from cdn

**Additional context**
When mouser over to any bar, all border lines displayed when hover.
Just 1 more question, can we turn on/off this hover animation ? I just like the straight line instead of the curved line of the bars. (already search the document but dont see how to do it).

Hoping hear your new soon. Thanks.

**Comment by martynasma** [comment]
This issue was fixed in dev version, and will be released soon.

We'll notify you here when that happens.

The answer to your other question is this:

```
series.slices.template.states.getKey("hover").properties.expandDistance = 0
```

However, I would prefer if you would post questions, not directly related to the original question/issue in a separate thread moving forward.

**Comment by PongVL** [comment]
Thanks for your help, do you need me bring it to a new question with your answer to easy trackback?

**Comment by martynasma** [comment]
> Thanks for your help, do you need me bring it to a new question with your answer to easy trackback?

Nah, unless you want to follow up on it.

**Comment by martynasma** [comment]
Fixed in [4.10.18](https://github.com/amcharts/amcharts4/releases/tag/4.10.18).

## [4.10.18] - 2021-04-21

### Fixed
- `SliceGrouper` was not working when chart was a child of another `Container`.
- `SliceGrouper` was not updating properly when `am4core.options.queue = true` was set.
- `DateAxis` labels were showing zeros instead of proper milliseconds in IE.
- In some cases hidden link in `SankeyDiagram` could cause rollover tooltip to appear.
- Stroke of the slices in a `FunnelChart` was not being drawn complete.
- JSON config: value list of the `List` element was not being properly truncated to supplied lenght.
- JSON config: event list as array was not working properly.
- Using `timezone` setting in some timezones could offset Series' bullets.
- Absolute pixel `width` in `RadarColumn` was being ignored.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
